File: src/main/java/com/kmarzecki/communicator/CommunicatorApplication.java
package com.kmarzecki.communicator;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@SpringBootApplication
@EnableTransactionManagement
public class CommunicatorApplication {

	public static void main(String[] args) {
		SpringApplication.run(CommunicatorApplication.class, args);
	}

}

File: src/main/kotlin/com/kmarzecki/trollo/CommunicatorApplication.kt
package com.kmarzecki.trollo

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.module.kotlin.KotlinModule
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean

@SpringBootApplication
class TrolloApplication {
    @Bean
    fun mapper(): ObjectMapper {
        return ObjectMapper().registerModule(KotlinModule())
    }
}

fun main(args: Array<String>) {
    runApplication<TrolloApplication>(*args)
}